tests/jballerina-unit-test/src/test/resources/test-src/typedefs/type-definitions-cyclic.bal

```diff
@@ -196,9 +197,19 @@ function testCastingToImmutableCyclicUnion() {
     (MyCyclicUnion & readonly)|error b = trap <MyCyclicUnion & readonly> a;
     assert(true, b is error);
     error err = <error> b;
-    assert("{ballerina}TypeCastError", err.message());
+    assert("{ballerina/lang.runtime}TypeCastError", err.message());
+    var detailMessage = err.detail()["message"];
+    string detailMessageString = detailMessage is error? detailMessage.toString(): detailMessage.toString();
```

Comment:
shall we format this properly in all places?
```suggestion
    string detailMessageString = detailMessage is error ? detailMessage.toString() : detailMessage.toString();
```
Let's properly format the entire file

Reply:
formatted